src/Service/Route.php

```diff
@@ -162,6 +162,7 @@ public function update(int $routeId, RouteUpdate $route, UserContext $context):
             $record = new Table\Generated\RoutesRow([
                 Table\Generated\RoutesTable::COLUMN_ID => $existing->getId(),
                 Table\Generated\RoutesTable::COLUMN_PRIORITY => $priority,
+                Table\Generated\RoutesTable::COLUMN_PATH => $route->getPath(), // Set Path to Update
```

Comment:
Hi @subangkit thanks for the PR, so in general the path was intentionally removed since changing a path of an existing route has a dramatic effect on every user who consumes the API, which could also breaks clients. Because of this we recommend to create a new route and deprecate the existing route instead of changing an existing route. Would this approach also work for your case?

Reply:
Hi @chriskapp , Got it. 
For better client's app integrity maybe I can suggest creating a manageable environment for example (development and production). Because for the development phase, creating a new route and able to updating path is needed. If there is some environmental management, the development process will be separately, after development finished we can promote to production ( which means We knowing the API is changing ).